### PR TITLE
lxqt-config-notificationd.desktop.in: fix QA errors

### PR DESCRIPTION
**Pull request by 10ne1**
Keys extending the standard format should start with "X-"
as per the desktop entry spec.

Signed-off-by: Ioan-Adrian Ratiu <adi@adirat.com>

### CODE REVIEW

**Comment by pmattern** [comment]
This PR is proposing to replace well defined *keys* with custom ones. This is just plain wrong.
What could be and has already been discussed is introducing a *value* `X-LXQt` upstream / in LXQt itself.

In keys `{Only,Not}ShowIn` arbitrary values are already allowed, this just hasn't been implemented in binary `desktop-file-validate` and the specification's corresponding [Appendix](http://standards.freedesktop.org/menu-spec/latest/apb.html) yet, see https://github.com/lxde/lxqt/issues/155#issuecomment-43912849.
In key `Categories` a key `X-LXQt` could be discussed. But introducint that value meant that it had to be replaced everywhere which is quite a task. All in all it seems more reasonable to try and get value `LXQt` registered which also seems justified given what's already [registered](http://standards.freedesktop.org/menu-spec/latest/apas02.html) by now.

Also note that right now those error messages thrown by `desktop-file-validate` and rightly bothering the package maintainers are the only problem. From a strictly technical point of view there's not a single problem due to value `LXQt` not being registered.
In other words the only problem triggering that whole discussion about `X-LXQt` and whatnot is a false-positive warning of a single binary - quite an absurd situation, basically.

All in all those values `LXQt` vs. `X-LXQt` are for sure something we need to address. But simply replacing `LXQt` with `X-LXQt` everywhere only for the only purpose of silencing a single binary's warnings doesn't seem like the right thing to do.

**Comment by 10ne1** [comment]
Thank you for the feedback, I'm dropping this pull request.
